cg/cli/workflow/rnafusion/base.py

```diff
@@ -64,37 +57,31 @@ def dev_config_case(cg_config: CGConfig, case_id: str):
 
 @rnafusion.command()
 @ARGUMENT_CASE_ID
-@OPTION_REVISION
-@OPTION_STUB
 @click.pass_obj
-def dev_run(cg_config: CGConfig, case_id: str, stub_run: bool, revision: str | None = None):
+def run(cg_config: CGConfig, case_id: str):
     """Run a preconfigured RNAFUSION case."""
     factory = AnalysisStarterFactory(cg_config)
     analysis_starter: AnalysisStarter = factory.get_analysis_starter_for_workflow(
         Workflow.RNAFUSION
     )
-    analysis_starter.run(case_id=case_id, stub_run=stub_run, revision=revision)
+    analysis_starter.run(case_id=case_id)
 
 
 @rnafusion.command()
 @ARGUMENT_CASE_ID
-@OPTION_REVISION
-@OPTION_STUB
 @click.pass_obj
-def dev_start(
-    cg_config: CGConfig, case_id: str, stub_run: bool = False, revision: str | None = None
-):
+def start(cg_config: CGConfig, case_id: str):
     """Start an RNAFUSION case. Configures the case if needed."""
     factory = AnalysisStarterFactory(cg_config)
     analysis_starter: AnalysisStarter = factory.get_analysis_starter_for_workflow(
         Workflow.RNAFUSION
     )
-    analysis_starter.start(case_id=case_id, stub_run=stub_run, revision=revision)
+    analysis_starter.start(case_id=case_id)
 
 
 @rnafusion.command()
 @click.pass_obj
-def dev_start_available(cg_config: CGConfig) -> None:
+def start_available(cg_config: CGConfig) -> None:
```

Comment:
Do we do return types for CLI commands? It looks like we don't so you can skip it here